### PR TITLE
feat(discord): add discord guideline on website

### PR DESCRIPTION
**Pull request by MadRex2000**
issue#875

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
Add discord guideline page on website.
And it can display the user's own token on the page.

## Steps to Test This Pull Request
1. Go to `localhost:8000/<lang>/ext/discord` by web browser.
2. Or use `localhost:8000/<lang>ext/discord?token={ token }` by web browser.

## Related Issue
issue #875 

## More Information
without token
截圖 2020-08-24 下午7 40 14

with token
截圖 2020-08-24 下午7 39 46

### CODE REVIEW

File: src/templates/pycontw-2020/ext/discord.html [hunk 1]
```diff
@@ -7,27 +7,44 @@
...
-
   <h2>{% trans 'discord-tutorials' %}</h2>
   <strong>{% trans "If you have not installed Discord or joined Python Taiwan Discord server, follow <a href='#setup-discord'>Setting up Discord</a> and setup your Discord first." %}</strong>
 
   <h3>{% trans '1. Register yourself' %}</h3>
   <p>{% trans 'Users can find a #registration-desk channel under <strong>PYCON TW 2020</strong> category. There is a PyCon TW RegBot in this channel to help you register to your roles.' %}</p>
   {% if token %}
-  <li>{% trans 'Enter' %} <strong>!register {{ token }}</strong> {% trans 'in this channel to register' %}</li>
+
+  <li>{% trans 'Enter' %} <code><a href="javascript:void(0)" id="copytoken" data-clipboard-text="!register {{ token }}">!register {{ token }}</a></code> {% trans 'in this channel to register' %}</li>
```
Comment (yychen) [inline src/templates/pycontw-2020/ext/discord.html:43]: I think it is a little bit not intuitive to know that this link serves as the copy function. Maybe put a `copy` icon? The base templates have font-awesome 4.7.0 included, so I suppose you can put something like this: https://fontawesome.com/v4.7.0/icon/clone

File: src/ext2020/views.py [hunk 2]
```diff
@@ -36,3 +36,7 @@ def live(request):
         'rooms': rooms,
         'token': token,
     })
+
+def discord(request):
+    token = request.GET.get('token', '')
+    return render(request, 'ext/discord.html', {'token': token})
```
Comment (yychen) [inline src/ext2020/views.py:42]: Should we do something if someone accidentally hit this page without a token? hmm...

**Review by yychen — COMMENTED** [review]
I think the overall styles need some tuning

File: src/ext2020/views.py (hunk 2 above)
Comment (MadRex2000) [reply src/ext2020/views.py:42]: If someone hit this page without token will display `!register {{ token }}`. 
Or are you mean that we should remind user need token to register ?

File: src/templates/pycontw-2020/ext/discord.html (hunk 1 above)
Comment (MadRex2000) [reply src/templates/pycontw-2020/ext/discord.html:47]: Like this (?
截圖 2020-08-26 上午12 52 26

Comment (yychen) [reply src/templates/pycontw-2020/ext/discord.html:47]: yes! like this. And I think you can add some padding after the second 輸入.

File: src/ext2020/views.py (hunk 2 above)
Comment (MadRex2000) [reply src/ext2020/views.py:42]: OK I get it.
The contents is temporarily I think we need to ask the program team.
截圖 2020-08-26 上午1 29 48

File: src/templates/pycontw-2020/ext/discord.html (hunk 1 above)
Comment (MadRex2000) [reply src/templates/pycontw-2020/ext/discord.html:47]: Roger that
截圖 2020-08-26 上午1 38 03

**Review by yychen — APPROVED** [review]
Let's merge this first and then take some time to see what's wrong with semantic ui.

**Pull request merged**